main/vstring.c

```diff
@@ -107,7 +107,7 @@ static void stringCat (
 	if (string->length + length + 1 > string->size)
 		vStringResize (string, string->length + length + 1);
 
-	strncpy (string->buffer + string->length, s, length);
+	memcpy (string->buffer + string->length, s, length);
```

Comment:
This *feels* dangerous, but yeah I searched through the code and the only thing that looked like it might have a behavior change with this is `readLine()`, which does this:

```
		if (File.allLines)
			vStringCat (File.allLines, File.line);
```

...and it got that `File.line` from `fgets()` which I believe does copy NUL bytes from the file... but the change in behavior *should* be benign. 🤞